sonar-orchestrator/src/main/java/com/sonar/orchestrator/build/ScannerForMSBuildExecutor.java

```diff
@@ -53,8 +58,20 @@ BuildResult execute(ScannerForMSBuild build, Configuration config, Map<String, S
     }
   }
 
-  private static Command createCommand(ScannerForMSBuild build, Map<String, String> adjustedProperties, File runnerScript) {
-    Command command = Command.create(runnerScript.getAbsolutePath());
+  private static Command createCommand(ScannerForMSBuild build, Configuration config, Map<String, String> adjustedProperties, File runnerScript) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused method parameter "config". ([squid:S1172](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1172))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.orchestrator%3Aorchestrator-parent&pullRequest=45&issues=AWMruZAtau6S9RpGg6Z_&open=AWMruZAtau6S9RpGg6Z_)

[ISSUE_KEY]: AWMruZAtau6S9RpGg6Z_